server/proxy/proxy.go

```diff
@@ -98,6 +100,104 @@ func normalizeStaticPath(conf config.Config, appID apps.AppID, icon string) (str
 	return icon, nil
 }
 
+// cleanForm removes:
+// - Fields without a name
+// - Fields with labels (either natural or defaulted from names) with more than one word
+// - Fields that have the same label as previous fields
+// - Invalid select static fields and their invalid options
+func (p *Proxy) cleanForm(form *apps.Form) {
+	if form == nil {
+		return
+	}
+
+	toRemove := []int{}
+	usedLabels := map[string]bool{}
+	for i, field := range form.Fields {
+		if field.Name == "" {
+			toRemove = append([]int{i}, toRemove...)
```

Comment:
Why is the order reversed?

Reply:
So we can then remove the elements from the list without any problem in the next for. We could do a normal descending for instead of a range and it will probably be better performant, but the code would be less clear (I think).

Reply:
Got it. Makes sense :+1: Maybe add a comment?